feature: added join

A1exCart3r opened the pull request:
Лаб. Раб 1. Задание 1. В.5
https://github.com/alexey-malov/oop/tree/master/tasks/01#%D0%B2%D0%B0%D1%80%D0%B8%D0%B0%D0%BD%D1%82-5--join--50-%D0%B1%D0%B0%D0%BB%D0%BB%D0%BE%D0%B2-

A1exCart3r committed:
feature: added join

https://github.com/alexey-malov/oop/tree/master/tasks/01#%D0%B2%D0%B0%D1%80%D0%B8%D0%B0%D0%BD%D1%82-5--join--50-%D0%B1%D0%B0%D0%BB%D0%BB%D0%BE%D0%B2-

A1exCart3r committed:
feature: join - fix to tests.bat

RuprechtKraus commented on join/tests.bat:
Если файл какой-то пустой

A1exCart3r replied:
добавил

RuprechtKraus commented:
В целом претензии что и в других лабах

A1exCart3r committed:
feature: join v2 added

Задание https://github.com/alexey-malov/oop/tree/master/tasks/01#%D0%B2%D0%B0%D1%80%D0%B8%D0%B0%D0%BD%D1%82-5--join--50-%D0%B1%D0%B0%D0%BB%D0%BB%D0%BE%D0%B2-
